input/v1beta1/resources_patches.go

```diff
@@ -37,9 +37,13 @@ type ToFieldPathPolicy string
 
 // ToFieldPath patch policies.
 const (
-	ToFieldPathPolicyReplace     ToFieldPathPolicy = "Replace"
-	ToFieldPathPolicyMergeObject ToFieldPathPolicy = "MergeObject"
-	ToFieldPathPolicyAppendArray ToFieldPathPolicy = "AppendArray"
+	ToFieldPathPolicyReplace                       ToFieldPathPolicy = "Replace"
+	ToFieldPathPolicyMergeObject                   ToFieldPathPolicy = "MergeObject" // Deprecated in favor of MergeObjects
+	ToFieldPathPolicyAppendArray                   ToFieldPathPolicy = "AppendArray" // Deprecated in favor of ForceMergeObjectsAppendArrays
+	ToFieldPathPolicyMergeObjects                  ToFieldPathPolicy = "MergeObjects"
+	ToFieldPathPolicyForceMergeObjectsAppendArrays ToFieldPathPolicy = "ForceMergeObjectsAppendArrays"
+	ToFieldPathPolicyForceMergeObjects             ToFieldPathPolicy = "ForceMergeObjects"
+	ToFieldPathPolicyMergeObjectsAppendArrays      ToFieldPathPolicy = "MergeObjectsAppendArrays"
```

Comment:
```suggestion
	ToFieldPathPolicyReplace                       ToFieldPathPolicy = "Replace"
	ToFieldPathPolicyMergeObjects                  ToFieldPathPolicy = "MergeObjects"
	ToFieldPathPolicyMergeObjectsAppendArrays      ToFieldPathPolicy = "MergeObjectsAppendArrays"
	ToFieldPathPolicyForceMergeObjectsAppendArrays ToFieldPathPolicy = "ForceMergeObjectsAppendArrays"
	ToFieldPathPolicyForceMergeObjects             ToFieldPathPolicy = "ForceMergeObjects"


	ToFieldPathPolicyMergeObject                   ToFieldPathPolicy = "MergeObject" // Deprecated: Use MergeObjects, which is functionally identical.
	ToFieldPathPolicyAppendArray                   ToFieldPathPolicy = "AppendArray" // Deprecated: Use ForceMergeObjectsAppendArrays, which is functionally identical.	
```

Nit: Sorts these as replace, regular merge, force merge, deprecated stuff.

Also uses the https://go.dev/wiki/Deprecated comment notation. Though I'm not 100% sure that works as a trailing comment.

Reply:
Thanks, addressed.